edsdme/blocks/search/SearchCards.js

```diff
@@ -40,7 +40,7 @@ export default class Search extends PartnerCards {
       return html`${repeat(
         this.paginatedCards,
         (card) => card.id,
-        (card) => html`<search-card class="card-wrapper" .data=${card} .localizedText="${this.blockData.localizedText}"></search-card>`,
+        (card) => html`<search-card class="card-wrapper" .data=${card} .localizedText="${this.blockData.localizedText} .ietf=${this.blockData.ietf}></search-card>`,
```

Comment:
It is still broken because of unclosed quotes in localizedText, it should be fine if we remove it from here
this is the test page: https://mwpw-153151-announcement-date--dme-partners--adobecom.hlx.page/channelpartners/drafts/dragana/search